### PR TITLE
First milestone feature

### PR DESCRIPTION
**Pull request by RICKCOYL**
- This is an oop project to create the game of tic tac toe.

- For the first milestone, all the setup files are created.

- Proper git-flow was followed.

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -1 +1 @@
-puts "Hello world!"
+puts 'Hello world!'
```
Comment (krys2fa) [inline bin/main.rb]: - [x] Please ensure that main.rb is an executable file.
You can do this by adding `#!/usr/bin/env ruby` to the top of the file.

- [x] Also, ensure executable file works (i.e. `bin/main.rb` should display the same message as `ruby bin/main.rb` [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Nicely done @ajkacca457 and @RICKCOYL  👏 
Your submission for this milestone is almost perfect. 💪 

### Status: Changes Required ♻️ 
Just a few more adjustments to complete the milestone. They can be seen in the comments below.

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

File: bin/main.rb (hunk 1 above)
Comment (ajkacca457) [reply bin/main.rb]: Thank you for your review and also for your help last night. We have ensured that the main.rb file is an executable file. We have also made sure that the command for executable file works.

File: bin/main.rb [hunk 2]
```diff
@@ -1 +1,2 @@
-puts "Hello world!"
+#!/usr/bin/env ruby
+puts '"Hello world!"'
```
Comment (bertil291utn) [inline bin/main.rb:2]: ## Leave simple quotes
Change from `puts '"Hello world!"'` to `puts 'Hello world!'`

**Review by bertil291utn — APPROVED** [review]
# **Status:** Approved ✅

## Hey guys, you've been doing a great job, and also I hope you enjoyed coding.



<h2 align="center">Ship it </h2>

<p align="center">image</p>

Please merge to develop branch and move on with the next milestone

You're doing  a great job keep going 💻 🚀 🔥

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Pull request merged**